index.js

```diff
@@ -1,4 +1,6 @@
 // load the dependencies
+var request = require('request');
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['request' is defined but never used.](https://www.codacy.com/app/jeffhann/git-mirror-sync/file/757207656/issues/source?bid=2446500&fileBranchId=2449083#l2)